portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/MyFormTests.testcase

```diff
@@ -0,0 +1,126 @@
+@component-name = "portal-web-forms-and-data-lists"
+definition {
+
+	property testray.main.component.name = "Forms";
+
+	setUp {
+		TestCase.setUpPortalInstance();
+
+		User.firstLoginPG();
+
```

Comment:
:yellow_circle: Empty line, as we talked some minutes ago, you need to execute SF after make changes